ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -133,392 +139,351 @@ Control {
-                            //TODO TBD, we need to show the number if the user has more than 1 of each collectible
-                            //not sure how to name the role
-                            visible: (model.userHas > 1)
-
-                            background: Rectangle {
-                                radius: 30
-                                color: Theme.palette.indirectColor2
+                        TapHandler {
+                            acceptedButtons: Qt.LeftButton | Qt.RightButton
+                            onSingleTapped: {
+                                if ((eventPoint.event.button === Qt.LeftButton) && (model.communityId !== "")) {
+                                    Global.openPopup(visitComunityPopupComponent, {communityId: model.communityId, communityName: model.communityName,
+                                                                                   communityLogo: model.communityImage, tokenName: model.name,
+                                                                                   tokenImage: model.imageUrl, isAssetType: false });
+                                } else {
+                                    if (eventPoint.event.button === Qt.LeftButton) {
+                                        Global.openLinkWithConfirmation(model.permalink, model.domain)
+                                    } else {
+                                        if (model.communityId !== "") {
+                                            Global.openMenu(delegatesActionsMenu, collectibleImage, { communityId: model.communityId, context: "collectibles"});
+                                        } else {
+                                            Global.openMenu(delegatesActionsMenu, collectibleImage, { url: model.permalink, domain: model.domain, context: "collectibles"});
+                                        }
+                                    }
+                                }
                             }
+                        }
+                        HoverHandler {
+                            id: hoverHandler
+                            cursorShape: hovered ? Qt.PointingHandCursor : undefined
+                        }
+                    }
 
-                            contentItem: StatusBaseText {
-                                id: amountText
-                                horizontalAlignment: Text.AlignHCenter
-                                verticalAlignment: Text.AlignVCenter
-                                font.pixelSize: Style.current.asideTextFontSize
-                                text: "x"+model.userHas
-                            }
+                    Image {
+                        id: gradient
+                        anchors.fill: collectibleImage
+                        visible: hoverHandler.hovered
+                        source: Style.png("profile/gradient")
+                    }
+
+                    //TODO Add drop shadow
```

Comment:
Is there a task to do it? 😄

Reply:
there is now https://github.com/status-im/status-desktop/issues/14194